BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/distance/BeaconDistanceCalculatorTest.java

```diff
@@ -31,4 +57,22 @@ public void calculateDistance_calibratedRssi_calibratedDistance() throws Excepti
         assertEquals(1, calculatedDistance, 0.1);
     }
 
+    public Beacon createDummyBeacon() {
+        Beacon dummyBeacon = new Beacon() {
+            @Override
+            public LocationProvider createLocationProvider() {
+                return null;
```

Comment:
Why do you create a new instance from an abstract class when you don't use it to create the LocationProvider anyway?